src/main/java/seedu/eduke8/ui/Ui.java

```diff
@@ -148,9 +152,7 @@ public void printQuestion(Question question, int questionNumber) {
     }
 
     public void printHint(Hint hint) {
-        System.out.println(HORIZONTAL_LINE);
-        System.out.println(MESSAGE_HINT + hint.getDescription());
-        System.out.println(HORIZONTAL_LINE);
+        printMessage(MESSAGE_HINT + hint.getDescription());
```

Comment:
Great change! We should have just use the method to print out messages as it already exists.